controllers/healthcheck.go

```diff
@@ -29,16 +30,16 @@ func HealthCheck(w http.ResponseWriter, r *http.Request) {
 
 	err := db.Exec("SELECT 1").Error
 	if err != nil {
-		log.Err(err).Msg("Could not ping database")
+		utils.LogErr(r, err, "Could not ping database")
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
 	w.WriteHeader(http.StatusOK)
 	_, err = w.Write([]byte("OK"))
 	if err != nil {
-		log.Err(err).Msg("Could not respond to HealthCheck")
+		utils.LogErr(r, err, "Could not respond to HealthCheck")
```

Comment:
Please add a `return` statement here so that the controller stops after logging the error.